src/Utilities/ProcessorArchitecture.cs

```diff
@@ -16,17 +16,17 @@ namespace Microsoft.Build.Utilities
     static public class ProcessorArchitecture
     {
         // Known processor architectures
-        public const string X86 = "x86";
-        public const string IA64 = "IA64";
+        internal const string X86 = "x86";
```

Comment:
We should probably not change the public surface area anyway, so leaving them public and documenting them is probably best.